rcljava/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ find_package(rcljava_common REQUIRED)
 find_package(rmw REQUIRED)
 find_package(rmw_implementation_cmake REQUIRED)
 find_package(rosgraph_msgs REQUIRED)
+find_package(rosidl_cmake)
```

Comment:
Is this change and addition to package.xml necessary? Seems unrelated to this PR.

Reply:
Build was failing without it, it didn't find `string_camel_case_to_lower_case_underscore()` (used [here](https://github.com/osrf/ros2_java/blob/5631e53191497320a4a8ee29e47701984ee7de32/rcljava/CMakeLists.txt#L90)), without it.
It seems that before we were getting that transitevely when find-packaging another package.